### PR TITLE
[GCI-142] renamed log with logger

### PR DESCRIPTION
**Pull request by Devanshk9**
## Description of what I changed
renamed log with logger
AS requested by sonar

## Issue I worked on
I worked on a sonar issue-https://sonar.openmrs.org/project/issues?id=org.openmrs%3Aopenmrs&issues=41211ce3-ea67-4148-b4e6-d7fe9e6f570c&open=41211ce3-ea67-4148-b4e6-d7fe9e6f570c

see https://issues.openmrs.org/browse/GCI-142

## Checklist: I completed these to help reviewers :)

- [ ] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by ribhavsharma** [comment]
Hi Devansh! I see that the Travis Build is failing. This is because of the fact that despite you renaming the variable, the rest of the file still uses the old variable name. Please replace those with the new one.
P.S The Refactor feature of Android Studio i.e ```Ctrl+Shift+R``` might help you ;)

**Comment by Devanshk9** [comment]
> Hi Devansh! I see that the Travis Build is failing. This is because of the fact that despite you renaming the variable, the rest of the file still uses the old variable name. Please replace those with the new one.
> P.S The _Find and Replace_ feature of Android Studio i.e `Ctrl+Shift+R` might help you ;)

Thnaks@ribhavsharma
done

**Comment by ribhavsharma** [comment]
The build is now passing :smile: Please squash you're commits and you should be good to go :)

File: api/src/main/java/org/openmrs/Concept.java [hunk 1]
```diff
@@ -70,57 +70,57 @@
  */
 @FullTextFilterDefs( { @FullTextFilterDef(name = "termsFilterFactory", impl = TermsFilterFactory.class) })
 public class Concept extends BaseOpenmrsObject implements Auditable, Retireable, Serializable, Attributable<Concept>,Customizable<ConceptAttribute> {
-	
+
 	public static final long serialVersionUID = 57332L;
-	
-	private static final Logger log = LoggerFactory.getLogger(Concept.class);
```
Comment (PrathameshMutkure) [inline api/src/main/java/org/openmrs/Concept.java]: Hii @Devanshk9, is your IDE properly configured to follow OpenMRS Code Style? I see unwanted indentation changes, wait for mentors to have a say on this.

Comment (Devanshk9) [reply api/src/main/java/org/openmrs/Concept.java]: @PrathameshMutkure check now

File: api/src/main/java/org/openmrs/Concept.java [hunk 2]
```diff
@@ -40,7 +40,7 @@
 import org.openmrs.api.db.hibernate.search.TermsFilterFactory;
 import org.openmrs.customdatatype.CustomValueDescriptor;
 import org.openmrs.customdatatype.Customizable;
-import org.openmrs.util.LocaleUtility;
+import org.openmrs.util.LocaleUtility;
```
Comment (riskycase) [inline api/src/main/java/org/openmrs/Concept.java]: Remove the trailing space here

File: api/src/main/java/org/openmrs/Concept.java [hunk 3]
```diff
@@ -60,7 +60,7 @@
  * fetch an object. To get child objects off of that Concept, further calls to the
  * {@link ConceptService} or the database are not needed. e.g. To get the list of answers that are
  * stored to a concept, get the concept, then call {@link Concept#getAnswers()}
- * 
+ *
```
Comment (riskycase) [inline api/src/main/java/org/openmrs/Concept.java]: Retain a trailing space after each * in a block comment

File: api/src/main/java/org/openmrs/Concept.java [hunk 4]
```diff
@@ -70,57 +70,57 @@
  */
 @FullTextFilterDefs( { @FullTextFilterDef(name = "termsFilterFactory", impl = TermsFilterFactory.class) })
 public class Concept extends BaseOpenmrsObject implements Auditable, Retireable, Serializable, Attributable<Concept>,Customizable<ConceptAttribute> {
-	
+
```
Comment (riskycase) [inline api/src/main/java/org/openmrs/Concept.java]: Restore the tab whitespace here and in other places as indicated by the diff

**Comment by riskycase** [comment]
Overall good work with the changes @Devanshk9, but please try to restrict your changes to fix the required issue only

**Pull request closed without merging**